content/applications/finance/accounting/fiscal_localizations/localizations/argentina.rst

```diff
@@ -75,56 +84,71 @@ Environment
+   request the certificate.
 
-   .. image:: argentina/argentina_edi_02.png
+   .. image:: argentina/request-certificate.png
+      :align: center
+      :alt: Request a certificate.
 
-#. Generate Certificate (AFIP). Access the AFIP portal and follow the instructions described in the
-   next document in order to get a certificate. `Get AFIP Certificate
-   <http://www.afip.gob.ar/ws/WSAA/wsaa_obtener_certificado_produccion.pdf>`_.
-   
-#. Upload Certificate and Private Key (Odoo). Once the certificate has been generated, it needs to
-   be uploaded in Odoo, using the pencil next in the field “Certificado” and selecting the
+#. :guilabel:`Generate Certificate (AFIP)`. Access the AFIP portal and follow the instructions
+   described in the next document to get a certificate. `Get AFIP Certificate
+   <https://drive.google.com/file/d/17OKX2lNWd1bjUt3NxfqcCKBkBh-Xlpo-/view>`_.
+
+#. :guilabel:`Upload Certificate and Private Key (Odoo)`. Once the certificate is generated, upload
+   it to Odoo using the **pencil icon** next to the field “Certificado” and select the
    corresponding file.
 
-   .. image:: argentina/argentina_edi_03.png
+   .. image:: argentina/upload-certificate-private-key.png
+      :align: center
+      :alt: Upload Certificate and Private Key.
 
 .. tip::
-   In case you need to configure the Homologation Certificate, please refer to the AFIP official 
+   In case you need to configure the Homologation Certificate, please refer to the AFIP official
    documentation: `Homologation Certificate
-   <http://www.afip.gob.ar/ws/documentacion/certificados.asp>`_.
+   <http://www.afip.gob.ar/ws/documentacion/certificados.asp>`_. Furthermore, Odoo allows the user
+   to test electronic invoicing locally without a Homologation Certificate. The following message
+   will be in the chatter when testing locally:
+
+   .. image:: argentina/local-testing.png
+      :align: center
+      :alt: Invoice validated locally because it is in a testing environment without testing
+            certificate/keys.
 
 Partner
 ~~~~~~~
 
 Identification Type and VAT
 ***************************
 
-As part of the Argentinean localization, the document types defined by the AFIP are now available on
-the Partner form, this information is essential for most transactions. There are six identification
-types available by default:
+As part of the Argentinian localization, document types defined by the AFIP are now available
+on the Partner form. Information is essential for most transactions. There are six identification
+types available by default, as well as 32 inactive types.
 
-.. image:: argentina/argentina04.png
+.. image:: argentina/identification-types.png
    :align: center
+   :alt: A list of AR Localization Document Types in Odoo, as defined by AFIP.
 
 .. note::
    The complete list of Identification types defined by the AFIP is included in Odoo but only the
```

Comment:
Here is the complete sentence: 
The complete list of Identification types defined by the AFIP is included in Odoo but only the common ones are active.
@vbe-odoo @samueljlieber